app/components/autocomplete/slash_suggestion/index.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+
+import {getAutocompleteCommands} from 'mattermost-redux/actions/integrations';
+import {getAutocompleteCommandsList} from 'mattermost-redux/selectors/entities/integrations';
+import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
+
+import SlashSuggestion from './slash_suggestion';
+
+// TODO: Remove when all below commands have been implemented
+const COMMANDS_TO_IMPLEMENT_LATER = ['collapse', 'expand', 'join', 'open', 'leave', 'logout', 'msg', 'grpmsg'];
+const NON_MOBILE_COMMANDS = ['rename', 'invite_people', 'shortcuts', 'search', 'help', 'settings', 'remove'];
```

Comment:
I see a few of this that we could handle in mobile but we can implement them later

Reply:
Yep, that's why I put them there 😉. Wanted to get the initial structure of the feature in first.